### ISSUE TITLE
Rocket Lake support

### ISSUE THREAD

**Issue by robertop23** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#191]
Hello folks,
Who knows what I need to do to add Rocket Lake support to this efi? I have a new 11700k to replace my old 10600 but is not working

**Comment by gtrooper** [comment]
> Hello folks,
> 
> Who knows what I need to do to add Rocket Lake support to this efi? I have a new 11700k to replace my old 10600 but is not working

The only thing you need to do is to change the CPU-ID to a Comet Lake One !

**Comment by gtrooper** [comment]
In your config.plist, Under Kernel-Emulate add these Values
<img width="686" alt="Fake CPU ID" src="https://user-images.githubusercontent.com/43722469/141099718-4f5aac8a-275e-46d8-8d0e-43e23f18857d.png">

**Comment by gtrooper** [comment]
Did you manage to boot with these values ?

**Comment by robertop23** [comment]
Yes, it works. Unfortunately, my NVME gen 4 works at the same speed than a gen 3, even with the latest Bios, so I decided change the motherboard

**Comment by gtrooper** [comment]
> Yes, it works. Unfortunately, my NVME gen 4 works at the same speed than a gen 3, even with the latest Bios, so I decided change the motherboard

- [x] There is a special setting in the BIOS which enables PCIe 4.0 speeds in the 1st NVMe PCIe slot near the CPU... check if you have this enabled... you should clearly be able to reach PCIe 4.0 ssd speeds

**Comment by gtrooper** [comment]
> Yes, it works. Unfortunately, my NVME gen 4 works at the same speed than a gen 3, even with the latest Bios, so I decided change the motherboard

Did you manage to enable the Setting in the BIOS ?

**Comment by robertop23** [comment]
No, I don't have that board  anymore
